config.m4

```diff
@@ -217,7 +217,6 @@ if test "$MONGODB" != "no"; then
     bson-timegm.c \
     bson-utf8.c \
     bson-value.c \
-    bson-version.c \
```

Comment:
This is not correct and will make https://github.com/mongodb-labs/mongo-php-driver-prototype/pull/104/files broken.

Please include all files. If the file was renamed, please use the new name